dev/devicelab/bin/tasks/gradle_plugin_test.dart

```diff
@@ -13,8 +13,10 @@ String javaHome;
 String errorMessage;
 
 /// Runs the given [testFunction] on a freshly generated Flutter project.
-Future<void> runProjectTest(Future<void> testFunction(FlutterProject project)) async {
-  final Directory tempDir = Directory.systemTemp.createTempSync('flutter_devicelab_gradle_plugin_test.');
+Future<void> runProjectTest(
```

Comment:
nit: The formatting on these lines was deliberate. Do you mind reverting the autoformatter  here? It also makes the diff a little harder to read to tell what's actually been changed. 

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#in-defense-of-the-extra-work-that-hand-formatting-entails
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters

In general it's also easier to review PRs that are small, and are either entirely just refactoring and style fixes or actual logic changes. Mixing the two makes it hard to track which is which and means that a potential revert will undo more than it needs to.

Reply:
Thanks for pointing it out. Won't use the dart auto formatter 👍